Assignment 2.Rmd

```diff
@@ -95,6 +122,51 @@ Now practice with data from our class. Please create a **person-network** with t
 
 Once you have done this, also [look up](http://igraph.org/r/) how to generate the following network metrics: betweeness centrality and dregree. **Who is the most central person in the network?**
 
+```{r}
+library(tidyr)
+library(dplyr)
+library(tibble)
+library(stringr)
+library(igraph)
+library(reshape2)
+
+# Import data
+table <- read.table("hudk4050-classes.txt", header = TRUE, sep = ",")
+
+# Crop data
+classdata <- table[-c(1:2),c(20:26)]
+person_class <- gather(classdata, "QID", "CID", 2:7)
```

Comment:
You probably need to combine last name and first name into one column before gathering.